src/components/teacher/CreateTaskModal.tsx

```diff
@@ -136,6 +136,9 @@ export default function CreateTaskModal({
 				description,
 				reward,
 				due: Timestamp.fromDate(dueDate),
+				gcrCourseID: '',
```

Comment:
If this is strictly for creating a regular task then these should not be set to empty string, these fields should not exist in the first place

Reply:
modified addTask